kimchi/js/bindings.js

```diff
@@ -1889,43 +2031,203 @@ var caml_plonk_verifier_index_of_rust = function (x, affine_class) {
     srs,
     evals,
     shifts,
-    None,
+    caml_lookup_index,
   ];
 };
+
+// Provides: caml_opt_to_rust
+// Requires: caml_is_none, None
+var caml_opt_to_rust = function (caml_optional_value, to_rust) {
+  if (caml_is_none(caml_optional_value)) {
+    return undefined;
+  } else {
+    return to_rust(caml_optional_value);
```

Comment:
I would add a single sentence as a documentation to describe if the function `to_rust` should expect `x` in the case of `Some x`. A some value is represented as `[0, vx]` where `vx` is the actual value.
I think it should be `caml_optional_value[1]`.

Reply:
Source: https://v2.ocaml.org/manual/intfc.html#ss:c-concrete-datatypes